ravenframework/utils/InputData.py

```diff
@@ -151,6 +151,7 @@ def __init__(self):
     self.parameterValues = {}
     self.subparts = []
     self.value = ""
+    self.additionalInput = []
```

Comment:
It might be useful to add a comment that mentions that additionalInput is of type xml.etree.ElementTree.Element

Reply:
(assuming I am reading the rest of the code correctly and this is actually the case.)

Reply:
(Adding a comment to subparts would also be useful (And should have been done before).)